Swiftfin tvOS/Components/SplitLoginWindowView.swift

```diff
@@ -0,0 +1,94 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+import SwiftUI
+
+struct SplitLoginWindowView: View {
+
+    // MARK: - Loading State
+
+    private var isLoading: Bool
+
+    // MARK: - Content Variables
+
+    private var leadingTitle: String
+    private var leadingContentView: () -> any View
+    private var trailingTitle: String
+    private var trailingContentView: () -> any View
+
+    // MARK: - Body
+
+    var body: some View {
+        VStack {
+            HStack {
```

Comment:
This "custom navigation bar" should be used across the user selection, server selection, and user sign in screens. So, it should probably be broken out into its own view.

Reply:
Sounds good. Make this into a `View` or a `View Modifier` like how the iOS filters are handled?

Reply:
This is what I came up with. Looks the same but can be tacked onto any view we need it to including `SplitLoginWindowView`

```
import Defaults
import SwiftUI

struct NavigationBarBrandingModifier: ViewModifier {

    var isLoading: Bool?

    func body(content: Self.Content) -> some View {
        VStack {
            VStack(alignment: .trailing) {
                if let loading = isLoading, loading == true {
                    ProgressView()
                }
            }
            .frame(maxWidth: .infinity)
            .frame(height: 100)
            .overlay {
                Image(.jellyfinBlobBlue)
                    .resizable()
                    .aspectRatio(contentMode: .fit)
                    .frame(height: 100)
                    .edgePadding()
            }
            .padding(.bottom)

            content

            Spacer()
        }
    }
}
```